internal/cri/server/status.go

```diff
@@ -94,5 +95,32 @@ func (c *criService) Status(ctx context.Context, r *runtime.StatusRequest) (*run
 		}
 		resp.Info["lastCNILoadStatus"] = defaultStatus
 	}
+
+	intro, err := c.client.IntrospectionService().Server(ctx, &ptypes.Empty{})
+	if err != nil {
+		return resp, err
+	}
+	if len(intro.Deprecations) == 0 {
+		cond := &runtime.RuntimeCondition{
+			Type:   ContainerdHasNoDeprecationWarning,
+			Status: true,
+		}
+		resp.Status.Conditions = append(resp.Status.Conditions, cond)
+	} else {
+		for _, d := range intro.Deprecations {
+			cond := &runtime.RuntimeCondition{
+				Type:    ContainerdHasNoDeprecationWarning,
```

Comment:
(I assume that `Type` does not need to be unique here)

Reply:
I hope so?  Considering this is a list rather than a map.